src/Apps/Order/Routes/__tests__/Respond.test.tsx

```diff
@@ -0,0 +1,229 @@
+import { BorderedRadio, Button } from "@artsy/palette"
+import {
+  Buyer,
+  OfferOrderWithShippingDetails,
+  Offers,
+  OfferWithTotals,
+} from "Apps/__tests__/Fixtures/Order"
+import { ArtworkSummaryItemFragmentContainer } from "Apps/Order/Components/ArtworkSummaryItem"
+import { CreditCardSummaryItemFragmentContainer } from "Apps/Order/Components/CreditCardSummaryItem"
+import { OfferHistoryItemFragmentContainer } from "Apps/Order/Components/OfferHistoryItem"
+import { OrderStepper } from "Apps/Order/Components/OrderStepper"
+import { ShippingSummaryItemFragmentContainer } from "Apps/Order/Components/ShippingSummaryItem"
+import { TransactionDetailsSummaryItemFragmentContainer } from "Apps/Order/Components/TransactionDetailsSummaryItem"
+import { Input } from "Components/Input"
+import { MockBoot } from "DevTools"
+import { mount } from "enzyme"
+import moment from "moment"
+import React from "react"
+import { commitMutation as _commitMutation } from "react-relay"
+import { CountdownTimer } from "Styleguide/Components/CountdownTimer"
+import { RespondFragmentContainer as RespondRoute } from "../Respond"
+
+jest.mock("Utils/getCurrentTimeAsIsoString")
+
+const NOW = "2018-12-05T13:47:16.446Z"
+
+require("Utils/getCurrentTimeAsIsoString").__setCurrentTime(NOW)
+
+jest.mock("react-relay", () => ({
+  commitMutation: jest.fn(),
+  createFragmentContainer: component => component,
+}))
+
+const testOrder = {
+  ...OfferOrderWithShippingDetails,
+  stateExpiresAt: moment(NOW)
+    .add(1, "day")
+    .toISOString(),
+  lastOffer: {
+    ...OfferWithTotals,
+    createdAt: moment(NOW)
+      .subtract(1, "day")
+      .toISOString(),
+  },
+  offers: { edges: Offers },
+  buyer: Buyer,
+}
+
+let mockPushRoute: jest.Mock<string>
+let mockMediatorTrigger: jest.Mock<string>
+
+describe("Offer InitialMutation", () => {
+  const getWrapper = (extraOrderProps?) => {
+    return mount(
+      <MockBoot>
+        <RespondRoute
+          relay={{ environment: {} }}
+          router={{ push: mockPushRoute }}
+          mediator={{ trigger: mockMediatorTrigger }}
+          order={{
+            ...testOrder,
+            ...extraOrderProps,
+          }}
+        />
+      </MockBoot>
+    )
+  }
+
+  beforeEach(() => {
+    mockPushRoute = jest.fn()
+    mockMediatorTrigger = jest.fn()
+  })
+
+  it("renders", () => {
+    const component = getWrapper()
+    const input = component.find(Input)
+    expect(input.text()).toContain("Your offer")
+  })
+
+  it("Shows the stepper", () => {
+    const component = getWrapper()
+    const stepper = component.find(OrderStepper)
+    expect(stepper.text()).toMatch("RespondReview")
```

Comment:
Can we add an assertion here to make sure that `Respond` is the active step? We do a similar thing [here](https://github.com/artsy/reaction/blob/master/src/Apps/Order/Routes/__tests__/Shipping.test.tsx#L396).